arangod/StorageEngine/TransactionState.h

```diff
@@ -224,18 +231,18 @@ class TransactionState {
 
   ServerState::RoleEnum const _serverRole;  // role of the server
 
-  CollectionNameResolver const& _resolver;
```

Comment:
this might be required by users of TransactionState to resolve data-source names, e.g. IResearchVinewNode

Reply:
everything else usually just ueses the transaction::Methods interface directly